tajo-common/src/main/java/org/apache/tajo/SessionVars.java

```diff
@@ -145,7 +145,7 @@
 
   // for index
   INDEX_ENABLED(ConfVars.$INDEX_ENABLED, "index scan enabled", DEFAULT),
-  INDEX_SELECTIVITY_THRESHOLD(ConfVars.$INDEX_SELECTIVITY_THRESHOLD, "the selectivity threshold for index scan",
+  INDEX_SELECTIVITY_LIMIT(ConfVars.$INDEX_SELECTIVITY_LIMIT, "the selectivity limit for index scan",
```

Comment:
Oops, it seems there was a misunderstanding for me.
According to Hyunsik's comment(please refer to: https://github.com/apache/tajo/pull/848#discussion_r44101976)

> I agreed. Threshold may be too long for human to put on cli. Actually, there mainly are two kinds of suffixes: SIZE and LIMIT. SIZE is literally used to represent size matter. LIMIT has been used to represent some number to determine whether doing or not doing according to the number. It would be better to make them consistent.

I was focusing on the consistency and thought all the terms should be matched using only `LIMIT` and `SIZE`.